templates/documentation/sdks/security-best-practices.md

```diff
@@ -0,0 +1,55 @@
+
+* All the api.video client libraries are using the [Advanced Authentication](https://docs.api.video/reference/disposable-bearer-token-authentication) method. If you are writing your own API client wrapper, be sure to leverage the Advanced Authentication instead of the Basic Authentication concept.
+
+* While serving the [upload token](https://docs.api.video/vod/delegated-upload-tokens) to the frontend, make sure that you maintain the shortest TTL possible for the tokens. In order to make sure that token is not hijacked and abused.
+
+* Maintain a secure channel of communication between the front-end and backend at all times.
+
+* Do not store API keys in files inside your application’s source tree. If you store API keys in files, keep the files outside your application’s source tree to help ensure your keys do not end up in your source code control system. This is particularly important if you use a public source code management system such as GitHub.
+
+* Do not embed API keys directly in code. API keys that are embedded in code can be accidentally exposed to the public. For example, you may forget to remove the keys from code that you share. Instead of embedding your API keys in your applications, store them in environment variables or in files outside of your application’s source tree.
+
+* Review your code before publicly releasing it. Ensure that your code does not contain API keys or any other private information before you make your code publicly available.
+Delete unneeded API keys to minimize exposure to attacks.
+
+* Limit one API key pair's usage to a specific system of your platform backend. This limits the scope of each key. If an API key is compromised, you can delete or regenerate the impacted key without needing to update your other API keys.
+
+
+## Securing the API keys on the application
+
+If you are forced to store the API keys directly in your application for technical reasons, make sure that you follow the below recommendations.
+
+Consider obfuscating the API key within your front-end application. Obfuscation transforms the key into a form that isn't immediately readable. However, it's crucial to understand that obfuscation is not foolproof. It merely makes the task of key retrieval more challenging, but not impossible. Determined malicious actors with advanced skills can still de-obfuscate the key. Think of obfuscation as an added layer of security, not a standalone solution.
+
+There are multiple guides around the web on how to secure your frontend application. You can find more recommendations on securing sensitive data on various frameworks and languages below:
+
+* [React Native - Storing sensitive data](https://reactnative.dev/docs/security#storing-sensitive-info)
+
+* [Flutter - Obfuscate code](https://docs.flutter.dev/deployment/obfuscate)
+
+* [Android - App Security](https://developer.android.com/privacy-and-security/security-tips)
```

Comment:
I haven't read all the security recommandation but the one on Android is not about storing a key. It's about global security, permissions,...
So it is not really relevant here.